.storybook/utils.js

```diff
@@ -258,9 +257,30 @@ export function autoPropKnobs(properties, bridge) {
       }
     }
   });
+
   return binding;
 }
 
+export function theming(bridge) {
```

Comment:
@kylebuch8 This is the only bit I updated in this file, the rest must be prettier.